features/FxA-97-password-strength/README.md

```diff
@@ -41,6 +41,16 @@ We believe that these changes will result in us collecting only the data that be
 * Datadog Dashboard › FxA Content Server - Registration › Signup funnel
  * Slightly reduced signup rate
 
+As of 25th July 2016,
+prior to shipping the new UX,;
+the [Datadog password-strength dashboard](https://app.datadoghq.com/dash/67511/fxa-content-server---password-strength)
+gives the following percentages for these events:
+
+* Passwords that are too short: 10.6%
+* Passwords that are just letters and numbers: 21.1%
+* Passwords that are too common (aka BLOOMFILTER_HIT): 1.67%
+
```

Comment:
Should we add `bloomfilter misses` and `successful signups as % of submissions` here as well? An increase in bloomfilter misses and/or a decrease in `bloomfilter_hit` is probably a good sign.